psi4/src/psi4/libpsipe/pe_calc_handler_i.hh

```diff
@@ -0,0 +1,31 @@
+#ifndef LIBPSIPE_PE_CALC_HANDLER_H
```

Comment:
Please switch to `.h` extension. And avoid "psi" in dirname -- we've got so many already :-)

Reply:
Resolved in 893f823f7 and 01ae0e5ac.
The header file you mentioned was not being used and I just forgot to remove it...
I renamed the `libpsipe` directory to `libpe`.